packages/midt/package.json

```
@@ -6,7 +6,7 @@
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "git://github.com/mapspeople/midt.git"
+    "url": "git://github.com/MapsPeople/web-ui/tree/main/packages/midt.git"
```

```suggestion
    "url": "git://github.com/MapsPeople/web-ui.git"
```

With midt being part of the generic web-ui repo, I'd guess this is the git url to put here, right?

---

If we want to point the users to the `web-ui` only instead of the full path to the `midt`, then yes

---

The `midt` folder is part of the `web-ui` git repo, so there is no such thing as `midt.git`, but we could link to its folder as the homepage.